src/main/java/com/theelixrlabs/healthcare/validations/DoctorModelValidator.java

```diff
@@ -0,0 +1,21 @@
+package com.theelixrlabs.healthcare.validations;
+
+import com.theelixrlabs.healthcare.constants.MessageConstants;
+import com.theelixrlabs.healthcare.exceptionHandler.CustomException;
+import org.springframework.context.MessageSource;
+import org.springframework.stereotype.Component;
+
+@Component
+public class DoctorModelValidator {
+    private final MessageSource messageSource;
+
+    public DoctorModelValidator(MessageSource messageSource) {
+        this.messageSource = messageSource;
+    }
+
+    public void validateDoctorName(String doctorName) {
+        if (doctorName == null || doctorName.trim().isEmpty()) {
```

Comment:
you can use StringUtils to perform null or empty checks

Reply:
yes checking the validation with the help of StringUtils..